.github/actions/install-doc-dependencies/action.yaml

```diff
@@ -8,31 +8,49 @@ runs:
     - name: Install required dependencies
       shell: bash
       run: |
+        set -e
```

Comment:
for optimizing the build (speed + stability), it may be worth using github's `cache` : if we cache the content of `/var/cache/apt`, then will be more tolerent to network issues and install will be much faster (and possibly we don't need complicated retry script)